nextjs/src/app/(logged-in)/data-sources/inspect/[externalDataSourceId]/getGeocodingConfigFromGeographyColumn.tsx

```diff
@@ -0,0 +1,54 @@
+'use client'
+import { GeographyTypes } from '@/__generated__/graphql'
+import { HubAreaType } from '@/app/reports/[id]/politicalTilesets'
+
+export function getGeocodingConfigFromGeographyColumn(
+  geographyColumn: string,
+  geographyColumnType: GeographyTypes
+):
+  | {
+      type: 'AREA'
+      components: [
+        {
+          type: 'area_code'
+          field: string
+          value: ''
+          metadata: { lih_area_type__code: HubAreaType[] }
+        },
+      ]
+    }
+  | {} {
+  const geocodingConfig = {
+    type: 'AREA',
+    components: [
+      {
+        type: 'area_code',
```

Comment:
I think this should be `area`, not `area_code`, in order to match both Area names and Area codes

Reply:
Power users can manually change `area` to `area_code` in the JSON, if necessary.

Reply:
Nice spot, that was a typo from out chat I forgot to fix